samples/wfi32iot/src/clicks/ultralowpress.c

```diff
@@ -67,14 +68,14 @@ void ULTRALOWPRESS_getData(float data[])
     //Convert to decimal value
     decimal = ULTRALOWPRESS_2sCompToDecimal(APP_SENSORS_data.i2c.rxBuffWords[0]);
     //Scale the value to actual
-    data[0] = ( (decimal - ULTRALOWPRESS_B0) / ULTRALOWPRESS_B1 );
+    ulp_data->temperature = ( (decimal - ULTRALOWPRESS_B0) / ULTRALOWPRESS_B1 );
 
     //Send command to read pressure
     APP_SENSORS_writeReadWords(ULTRALOWPRESS_I2CADDR, ULTRALOWPRESS_REG_PRESS, 2);
     //Convert to decimal value
     decimal = ULTRALOWPRESS_2sCompToDecimal(APP_SENSORS_data.i2c.rxBuffWords[0]);
     //Scale the value to actual
-    data[1] = -20.0 + ( ( decimal + 26215.0 ) / ( 26214.0 + 26215.0 ) ) * (520);
+    ulp_data->pressure = -20.0 + ( ( decimal + 26215.0 ) / ( 26214.0 + 26215.0 ) ) * (520);

```

Comment:
Code style: Some extra spaces between brackets should be removed like
```c
ulp_data -> pressure = -20.0 + ((decimal + 26215.0) / (26214.0 + 26215.0)) * (520);
```
Same ssue few lines above. Maybe search in other files too.